IS7024FinalProjectBreweryArc/BreweryArc/wwwroot/css/site.css

```diff
@@ -69,3 +69,8 @@ body {
   white-space: nowrap;
   line-height: 60px; /* Vertically center the text there */
 }
+

```

Comment:
Added a style class which is used in the Index.cshtml page for better usability.